spring-batch-core/src/main/java/org/springframework/batch/core/step/builder/AbstractTaskletStepBuilder.java

```diff
@@ -74,6 +75,23 @@ public AbstractTaskletStepBuilder(StepBuilderHelper<?> parent) {
 		super(parent);
 	}
 
+	/**
+	 * Create a new builder initialized with any properties in the parent. The parent is
+	 * copied, so it can be re-used.
+	 * @param parent a parent helper containing common step properties
+	 */
+	public AbstractTaskletStepBuilder(AbstractTaskletStepBuilder<?> parent) {
+		super(parent);
+		this.chunkListeners = parent.chunkListeners;
+		this.stepOperations = parent.stepOperations;
+		this.transactionManager = parent.transactionManager;
+		this.transactionAttribute = parent.transactionAttribute;
+		this.streams.addAll(parent.streams);
```

Comment:
> Hmm maybe we can just use `parent.streams` instead of `addAll()`?

To creating a copy constructor, it seems like the second option, ```this.streams = parents.streams```, is appropriate. Thank you for your comment @injae-kim!

But just curious.. was there any specific reason for initially declaring ```streams``` as a ```final``` variable?